common/lib/xmodule/xmodule/fields.py

```diff
@@ -252,3 +253,19 @@ def enforce_type(self, value):
             return value
 
         return self.from_json(value)
+
+
+class ScoreField(JSONField):
+    def from_json(self, value):
+        if value is None:
+            return value
+        if isinstance(value, Score):
+            return value
```

Comment:
Any update on this? I wouldn't call it blocking, but it seems like weird behavior that could indicate bugs elsewhere.